wfcGen.py

```diff
@@ -164,85 +166,97 @@ def __init__(self, sx, sy, cells, tileSize=3):
 
     def setRandom(self):
         self.grid = []
+        self.entropySave = {}
         for y in range(self.sy):
             self.grid.append([])
             for x in range(self.sx):
                 self.grid[y].append(Cell([x, y]))
-        self.grid[random.randint(0, self.sy-1)][random.randint(0, self.sx-1)].setCell(random.randint(0, len(self.cells)-1), self.cells)
-        self.getFinalImage()
-
-    def loadEntropy(self):
-        self.entropySave = {}
-        for y in range(0, self.sy):
-            for x in range(0, self.sx):
-                self.grid[y][x].loadEntropy(self.grid)
-                if not self.grid[y][x].collapsed:
-                    entropy = self.grid[y][x].getEntropy(self.cells)
-                    self.entropySave[entropy] = self.entropySave.get(entropy, []) + [[x, y]]
+        rand = [random.randint(0, self.sy-1), random.randint(0, self.sx-1)]

```

Comment:
This variable does not make much sense. As far as I can see, you are only using `rand[0]` and `rand[1]` separately. So, why not just have two variables (with telling names!)?